networks.json

```diff
@@ -9,8 +9,8 @@
     "4": {
       "events": {},
       "links": {},
-      "address": "0xc2425e2B18cc3dF29BB9917bD3Cc7bCb4BCBde1A",
-      "transactionHash": "0x975468d49db21f7ea8056c30f014b1f634a1436874179553ebec771a3fff32ff"
+      "address": "0xC1BE999de97dE9EB87Ef119bAB9dc396e4Eaf70C",
```

Comment:
I'd prefer having deploys in separate PRs (no need to review those), but isolated.

Reply:
sounds good, I'll do that from now on

Reply:
sounds good, I'll do that from now on